### PR TITLE
[WIP] Test fix

### PR DESCRIPTION
**Pull request by ymesika**

### CODE REVIEW

File: tests/e2e/framework/multicluster.go [hunk 1]
```diff
@@ -105,6 +105,7 @@ func (k *KubeInfo) generateRemoteIstio(dst string, useAutoInject bool, proxyHub,
 			log.Infof("Endpoint for service %s not found", svc)
 		}
 	}
+	helmSetContent += " --set security.selfSigned=false"
```
Comment (utka) [inline tests/e2e/framework/multicluster.go]: Can you explain (add comment) why we need to have selfSigned false?

Comment (ymesika) [reply tests/e2e/framework/multicluster.go]: Sure. Once/If it's out of WIP

**Pull request closed without merging**